src/tests/ftest/io/fio_small.yaml

```diff
@@ -0,0 +1,54 @@
+hosts:
+    test_servers:
+        - server-A
+        - server-B
+        - server-C
+        - server-D
+    test_clients:
+        - client-E
+timeout: 1000
+server_config:
+    name: daos_server
+    servers:
+        bdev_class: nvme
+        bdev_list: ["0000:5e:00.0","0000:5f:00.0"]
+        scm_class: dcpm
+        scm_list: ["/dev/pmem0"]
+pool:
+    createmode:
+        mode_RW:
+             mode: 146
+    createset:
+        group: daos_server
+    createsize:
+        scm_size: 1600000000
+        nvme_size: 9000000000
+    createsvc:
+        svcn: 1
+fio:
+  global:
+    name: 'global'
+    directory: "/tmp/daos_dfuse"
+    ioengine: 'libaio'
+    thread: 16
```

Comment:
what does that mean (setting it to 16)?
AFAIK it's 1 or 0; use threads or use processes. unless that changed in a recent FIO release..